sql/src/test/java/io/crate/operation/collect/files/FileReadingCollectorTest.java

```diff
@@ -144,13 +144,15 @@ public void testRelativeImport() throws Throwable {
 
     @Test
     public void testCollectFromUriWithGlob() throws Throwable {
-        CollectingRowReceiver projector = getObjects(Paths.get(tmpFile.getParentFile().toURI()).toUri().toString() + "file*.json");
+        CollectingRowReceiver projector = getObjects(
+            Paths.get(tmpFile.getParentFile().toURI()).toUri().toString() + "file*.json");
         assertCorrectResult(projector.result());
     }
 
     @Test
     public void testCollectFromDirectory() throws Throwable {
-        CollectingRowReceiver projector = getObjects(Paths.get(tmpFile.getParentFile().toURI()).toUri().toString() + "*");
+        CollectingRowReceiver projector = getObjects(
+            Paths.get(tmpFile.getParentFile().toURI()).toUri().toString() + "*");
         assertCorrectResult(projector.result());
```

Comment:
pls add tests for multithreaded consume of the BatchIterator of the collector. probably with a multithreaded Consumer test implementation.